src/docker/docker.cpp

```diff
@@ -381,6 +381,8 @@ Future<Nothing> Docker::run(
   argv.push_back("-e");
   argv.push_back("MESOS_SANDBOX=" + mappedDirectory);
 
+  /*
+
```

Comment:
Remove the attempt to mount the Mesos sandbox as a host volume in the Docker container. Because Docker containers run in a multi-tenant environment, there's [no access to the underlying host filesystem](https://apidocs.joyent.com/docker/features/volumes). This is an important factor in multi-tenant security.